src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +49,212 @@ export const KafkaPage: FunctionComponent = () => {
+              >
+                {" "}
+                Redhat support
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
+              </Button>{" "}
+              {t("kafkaInstanceCapacityfooter2")}
+              <Button
+                data-testid="diskSpaceMetrics-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"
+              >
+                {" "}
+                monitoring disk space metrics
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
+              </Button>
+              .
+            </TextContent>
+          </CardFooter>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection kafkaOverviewPage={true}>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("cloudProvidersTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex spaceItems={{ default: "spaceItemsXl" }}>
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Stack>
+                    <StackItem>
+                      <img src={LogoAWS} alt={""} style={{ height: "60px" }} />
+                    </StackItem>
+                    <StackItem>
+                      <Title headingLevel="h2" className={"pf-u-pt-sm"}>
+                        {"Amazon Web Services"}
+                      </Title>
+                    </StackItem>
+                  </Stack>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Stack>
+                    <StackItem>
+                      <img
+                        src={LogoAzure}
+                        alt={""}
+                        style={{ height: "60px" }}
+                      />
+                    </StackItem>
+                    <StackItem>
+                      <Title headingLevel="h2" className={"pf-u-pt-sm"}>
+                        {"Microsoft Azure"}
+                      </Title>
+                    </StackItem>
+                    <StackItem>
+                      <Text
+                        component={TextVariants.p}
+                        className={"pf-u-color-200"}
+                      >
+                        {t("microsoftAzureDescription")}
+                      </Text>
+                    </StackItem>
```

Comment:
Above I mentioned some options available for fixing this layout. This section will probably require a bit more work that the AWS option. If you go with option one, you still might want to keep the text inside a stack layout so that the text is stacked.

If you want to go with option two, then I think everything can be a flex item, but you likely need to set smaller spacing between the two lines of text. You can see how to individually space elements in this [PF example](https://www.patternfly.org/v4/layouts/flex#individually-spaced).